api/testutils/loctest/helper.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package loctest
```

Comment:
+1. I like the suggestion. "helper" and "util" are always less preferred due to its vague meaning.

Reply:
I moved the `api` test helper changes to #4974: https://github.com/kubernetes-sigs/kustomize/pull/4959#issuecomment-1380826106

@natasha41575 renamed package!
@yuwenma I renamed the file to `filesystem.go` because all the test helpers deal with the real file system. Let me know if you have a better name in mind!